app/src/utils/utils.ts

```diff
@@ -36,3 +36,16 @@ export function parseCalendarJSON(data: ICourse[]): ICalendarItem[] {
   });
   return calendarData;
 }
+
+// Given a data source and a professor username, return courses that professor is teaching or has taught.
+export function getCoursesForProfessor(username: string, data: ICourse[]): ICourse[] {

```

Comment:
Should we move this into a "api" sort of file instead? Imo utils are for helper stuff not api calls. However I would want to keep this in utils if we need to format the data after the api call. In that case we can just add an api call to a different file later

Reply:
Definitely a good idea and something we can configure when we have a connection to hook up to later in the sprint (hopefully). Let's leave this here for now and I'll create a todo to move this filtering logic to our API service once it's implemented.

Reply:
Sounds good, adding in in my integration PR